webapp/src/dogma/features/project/settings/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Button,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@chakra-ui/react';
+
+interface DeleteConfirmationModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  type: string;
+  id: string;
+  projectName: string;
+  handleDelete: () => void;
+  isLoading: boolean;
+}
+
+export const DeleteConfirmationModal = ({
+  isOpen,
+  onClose,
+  id,
+  type,
+  projectName,
+  handleDelete,
+  isLoading,
+}: DeleteConfirmationModalProps): JSX.Element => {
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>Are you sure?</ModalHeader>
+        <ModalCloseButton />
+        <ModalBody>
+          Delete {type} &apos;{id}&apos; from {projectName}?
```

Comment:
It seems that the previous comment wasn't applied.
```suggestion
          Delete {type} '{id}' from {projectName}?
```

Reply:
<img width="484" alt="image" src="https://github.com/user-attachments/assets/121c67be-ea03-4faa-95e8-8cedf466106e">
It's actually working now. 😓 
I forgot to add `;` before.

Reply:
Side note: We can use `'` without encoding.

Reply:
It's just that IDE complaining about it. 🤣